src/App.tsx

```diff
@@ -54,6 +54,16 @@ const styles = StyleSheet.create({
     backgroundColor: PlatformColor('NavigationViewDefaultPaneBackground'),
     height: '100%',
   },
+  drawerTopDivider: {
```

Comment:
could we use a single style for both, or do you expect these to diverge down the road?

Reply:
Already have to diverge because one uses bottom and one used top for border width.